src/vmspawn/vmspawn.c

```diff
@@ -208,11 +239,163 @@ static int parse_argv(int argc, char *argv[]) {
         return 1;
 }
 
+static int open_vsock(void) {
+        int r;
+        _cleanup_close_ int vsock_fd = -EBADF;
+
+        r = socket(AF_VSOCK, SOCK_STREAM|SOCK_CLOEXEC, 0);
+        if (r < 0)
+                return log_error_errno(errno, "Failed to open AF_VSOCK socket: %m");
+        vsock_fd = r;
+
+        static const union sockaddr_union bind_addr = {
+                .vm.svm_family = AF_VSOCK,
+                .vm.svm_cid = VMADDR_CID_ANY,
+                .vm.svm_port = VMADDR_PORT_ANY,
+        };
```

Comment:
We tend to put static const definitions like this at the beginning the of function.

Reply:
Fixed in 084ed8ddc7.